<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>extInfo</class>
 <widget class="QDialog" name="extInfo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1276</width>
    <height>626</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>71</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Info for ID:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>113</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>256</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_2">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>310</y>
     <width>256</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_3">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>310</y>
     <width>256</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QTableView" name="tableView_4">
   <property name="geometry">
    <rect>
     <x>970</x>
     <y>310</y>
     <width>256</width>
     <height>301</height>
    </rect>
   </property>
  </widget>
  <widget class="QFrame" name="">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>60</y>
     <width>611</width>
     <height>178</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0" colspan="2">
     <widget class="QRadioButton" name="radioButton">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Pending Requests:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="comboBox"/>
    </item>
    <item row="1" column="4">
     <widget class="QComboBox" name="comboBox_2"/>
    </item>
    <item row="3" column="3" rowspan="2" colspan="2">
     <widget class="QCheckBox" name="chkPL">
      <property name="text">
       <string>Lenders</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3" colspan="2">
     <widget class="QCheckBox" name="chkPCurr">
      <property name="text">
       <string>Current Amount</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="2">
     <widget class="QCheckBox" name="chkLLen">
      <property name="text">
       <string>Lenders</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" colspan="2">
     <widget class="QCheckBox" name="chkLNum">
      <property name="text">
       <string>Number of  installments</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Loan:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0" rowspan="2" colspan="2">
     <widget class="QCheckBox" name="chkLRem">
      <property name="text">
       <string>Remaining Amount</string>
      </property>
     </widget>
    </item>
    <item row="6" column="0" colspan="2">
     <widget class="QCheckBox" name="chkLMed">
      <property name="text">
       <string>Medians</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2" colspan="2">
     <widget class="QRadioButton" name="radioButton_2">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
